Cython/Utility/Exceptions.c

```diff
@@ -482,17 +501,52 @@ static CYTHON_INLINE void __Pyx__ExceptionReset(PyThreadState *tstate, PyObject
 #define __Pyx_ExceptionReset(type, value, tb)  PyErr_SetExcInfo(type, value, tb)
 #endif
 
+/////////////// _ExceptionGetType.proto ///////////////
+
+static CYTHON_INLINE PyObject* __Pyx__ExceptionGetType(PyObject *value) {
```

Comment:
The naming convention is to use double underscores for internal variants of "regular" helper functions, e.g. after handling special cases in an inline function. These functions should just have a single underscore as name separator.